xorg.rb

```diff
@@ -63,6 +56,7 @@ class Xorg < Formula
   depends_on "linuxbrew/xorg/libpciaccess" => args_check_static
   depends_on "linuxbrew/xorg/libxkbfile" => args_check_static
   depends_on "linuxbrew/xorg/libxshmfence" => args_check_static
+  depends_on "linuxbrew/xorg/xcb-proto" => args_check_static
```

Comment:
I'm fine with adding this line, but I'm curious, did you find a formula that needs it?